iidm/iidm-api/src/main/java/com/powsybl/iidm/network/PhaseTapChangerHolder.java

```diff
@@ -30,5 +30,14 @@
     /**
      * Get the optional ratio tap changer.
      */
-    Optional<PhaseTapChanger> getOptionalPhaseTapChanger();
+    default Optional<PhaseTapChanger> getOptionalPhaseTapChanger() {
+        return Optional.ofNullable(getPhaseTapChanger());
+    }
+
+    /**
+     *  Check if a phase tap changer is present
+     */
+    default boolean hasPhaseTapChanger() {
+        return getPhaseTapChanger() != null;
```

Comment:
You can add an implementation in iidm-impl (even if it is the same behavior) using the field in the impl class rather than the `getPhaseTapChanger` method